funnel/assets/cypress/integration/11_confirmProposal.js

```diff
@@ -56,5 +57,12 @@ describe('Confirm proposal', function () {
     var cid = window.location.hash;
     cy.get(`${cid} .comment__body`).contains(proposal.comment);
     cy.get(`${cid} .comment__header`).contains(editor.username);
+    cy.visit('/');
+    cy.logout();
+    cy.wait(1000);
+    cy.login('/' + profile.title, member.username, member.password);
+    cy.visit('/updates');
+    cy.wait(1000);
+    cy.contains('You submitted');
```

Comment:
Add test to verify that comment has been received on the proposal. Also better to test for proposal title.
    cy.contains('You submitted');
    cy.contains(proposal.comment);